packages/discord.js/typings/index.d.ts

```diff
@@ -1089,7 +1089,7 @@ export class GuildMember extends PartialTextBasedChannel(Base) {
   public readonly displayName: string;
   public guild: Guild;
   public readonly id: Snowflake;
-  public pending: boolean;
+  public pending: boolean | null;
```

Comment:
Shouldn't this be changed in `PartialGuildMember` instead as it's null only when partial?
https://github.com/discordjs/discord.js/blob/50c6ddf1dbcfb5fe740c3ac13a9c85b0ece66819/packages/discord.js/typings/index.d.ts#L4773